latest/index.bs

```diff
@@ -331,24 +338,1010 @@ Conforming readers:
 - MAY ignore other groups or arrays under the root of the hierarchy.
 
 
+"coordinateSystems" metadata {#coord-sys-md}
```

Comment:
It would help to clarify exactly where this is specified within the zarr attribute structure. The example could include the top-level "ome" key to clarify.   If this isn't intended as a top-level member of the "ome" object, then perhaps it should be described in a later section, after describing its "parent" object, in order to make the specification clearer.

Reply:
I think this could be addressed in #138, which this PR is based on.